<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROIInfoWindow</class>
 <widget class="QMainWindow" name="ROIInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROI Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>ROI set</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblROISet">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>ROI type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lblROIType">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>ROI index</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lblROIIndex">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>ROI diameter</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="txtd1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>X (mm)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Y (mm)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Z (mm)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtXCenter"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtYCenter"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtZCenter"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Value</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="leValue"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pbUpdate">
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtT1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>T2 (ms)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtT2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>ADC*1000 (mm2/s)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txtADC"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Concentration (mM)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtConcentration"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>proton density (%)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="txtProtonDensity"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Ave signal</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lblAve"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>standard deviation</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lblSd"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string># voxels</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lenVoxels"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Column</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="leH"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Row</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="leV"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>T1 (ms)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ImageView" name="imvROI"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUpdate_Values">
    <property name="title">
     <string>Files</string>
    </property>
   </widget>
   <addaction name="menuUpdate_Values"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpdate_values">
   <property name="text">
    <string>Update values</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
